src/main.py

```
@@ -2,5 +2,47 @@ def main():
     print("Hello, World!")
 
+# Global variable with unclear name
+x = []
+
+# Function that does multiple unrelated things with poor naming
+def do_stuff(a,b,c,d):   # No spaces after commas
+    global x
+    
+    try:
+        result=a+b/c*d    # No spaces around operators
+    except:    # Bare except clause - bad practice
```

Ruff (0.8.2)

Do not use bare `except`

(E722)